### DISCUSSION TITLE
`mdx` set up failed

### DISCUSSION THREAD

**Discussion by winner106** [remotion-dev/remotion#2624] [General]
Could you please provide some more guidance steps? Thank you. @JonnyBurger 

what I have tried:

![image](https://github.com/remotion-dev/remotion/assets/50519642/909a7ff4-4fda-47f9-9960-7b6950ead1c9)

![image](https://github.com/remotion-dev/remotion/assets/50519642/0aaa7fc5-1839-4d0b-9f18-1390cc5e8d0f)

![image](https://github.com/remotion-dev/remotion/assets/50519642/30823019-0724-4894-b2f4-57df9851b82e)

![image](https://github.com/remotion-dev/remotion/assets/50519642/d2d7884c-6022-4add-b93f-2c201fbc5f9d)

preview:

![image](https://github.com/remotion-dev/remotion/assets/50519642/ad912440-5ba5-42a9-8a13-7ccdbe999100)

error:

```powershell
ERROR in ./src/remotion/projects/mdx/content.mdx 2:0-81
Module not found: Error: Can't resolve 'react/jsx-dev-runtime' in 'K:\next-remotion\src\remotion\projects\mdx'
```

**Comment by JonnyBurger** [comment]
Sorry for the late response!

For me, the MDX loading works in our testbed.

`react/jsx-dev-runtime` was introduced in React 17.0.
My suspicion is that your version of `react` is older, am I right with that?

**Reply by winner106** [reply]
I created a  totally new remotion project by  `pnpm create video` , I am sure the react version is 18.0.

Then I followed [the instructions in the document](https://www.remotion.dev/docs/webpack#enabling-mdx-support) to configure `enable mdx`, but I still get an error when I run it. 

I have been searching for various information for the past few days but have not been able to solve it. It's really frustrating. I hope you can help me take a look at where the problem might be when you have time. Thank you very much. @JonnyBurger 

I also tried the new version of  [@mdx-js/loader](https://mdxjs.com/packages/loader/) , but it still failed.

new remotion project: [https://github.com/winner106/remotion-mdx](https://github.com/winner106/remotion-mdx)

**Comment by JonnyBurger** [comment]
Thanks for providing the repro!
I figured out we also need to add `jsx-dev-runtime` to Webpack aliases for the new MDX loader to work. Doing this in #2664 and also updating the docs so they work with the newest MDX version.

**Reply by winner106** [reply]
Thank you so much! I'm looking forward to it as well.
